docs/samples/Microsoft.ML.Samples/Dynamic/WordEmbeddingTransform.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Text;
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class WordEmbeddingTransform
+    {
+        public static void ExtractEmbeddings()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable and convert to IDataView.
+            var data = SamplesUtils.DatasetUtils.GetSentimentData();
+            var trainData = ml.Data.ReadFromEnumerable(data);
+
+            // Preview of the data.
+            //
+            // Sentiment    SentimentText
+            // true         Best game I've ever played.
+            // false        ==RUDE== Dude, 2.
+            // true          Until the next game, this is the best Xbox game!
+
+            // Pipeline which goes through SentimentText and normalize it, tokenize it by words, and removes default stopwords.
+            // After all that cleaning, we apply WordEmbedding with GloVeTwitter25D.
+            // 25D means each word mapped into 25 dimensional space, basically each word represented by 25 float values.
+            var wordsPipeline = ml.Transforms.Text.NormalizeText("NormalizedText", "SentimentText")
```

Comment:
Recommend just using the normal text transform w/ `tokens=+`, which will do this for the user.

Reply:
Do you think we should change `TextTransform{tokens=+}` to output the tokens column named `tokens`?

I've always found `{outputcolumn}_OutputTokens` rather confusing, and different from the normal behavior.